packages/vega-scenegraph/src/util/text.js

```diff
@@ -130,9 +130,12 @@ export function offset(item) {
   // why? not all browsers support SVG 1.1 'alignment-baseline' :(
   var baseline = item.baseline,
       h = fontSize(item);
+
   return Math.round(
+    baseline === 'line-top' ? 0.79.h + (lineHeight(item)-h) / 2.0 :
```

Comment:
I am fixing this myself, but note the `0.79.h` (which should be `0.79*h`), which suggests that this code has never been evaluated, much less tested.

Reply:
sorry about that, I forgot to mark the PR as WIP (as I was doing it while traveling).